stylesheets/style.css

```diff
@@ -0,0 +1,219 @@
+body {
+  padding: 0;
+  margin: 0;
+  background: #eceef1;
+}
+
+.sign-in-info {
+  float: right;
+  margin-right: 45px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  margin: -140px;
+  margin-right: 25px;
+}
+
+.common-styles {
+  font-size: 12px;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.forgot-password {
+  margin-top: -12px;
+  color: #0077c5;
+}
+
+.forgot-password a {
+  text-decoration: none;
+}
+
+.signin-btn {
+  margin-top: 12px;
+  margin-left: 12px;
+}
+
+.signin-btn button {
+  height: 35px;
+  width: 90px;
+  border-radius: 4px;
+  background-color: #0077c5;
+  color: white;
+  margin-top: -2px;
+  border: 1px solid;
+}
+
+.signin-btn i {
+  font-size: small;
+}
+
+.sign-up {
+  margin-top: 25px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.form-size {
+  height: 695px;
+  width: 400px;
+  background-color: white;
+  border: 1px solid gray;
+  position: relative;
+}
+
+.logo img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 150px;
```

Comment:
[OPTIONAL] You could use a shorthand notation for margin and padding. In this case, for margin, the same code could be re-written as: `margin: 150px auto 0;` , where 150px is the top margin, auto applies to both right and left margins, and 0 is for the bottom one.

Note: If you use four values for `margin` it applies to all sides. More info [here](https://css-tricks.com/almanac/properties/m/margin/)